tests/cross_functional/kcs/test_selinux_relabel_solution.py

```diff
@@ -267,9 +267,9 @@ def test_selinux_relabel_for_existing_pvc(
         self.pod_selector = self.pod_obj.labels.get(constants.DEPLOYMENTCONFIG)
 
         # Leave pod for some time to run since file creation time is longer
-        waiting_time = 120
+        waiting_time = 240
```

Comment:
Can we use timeoutsampler/retry method polling some resource here, instead of the hardcoded sleep time?